test/verify/check-machines-dbus

```diff
@@ -58,6 +58,32 @@ class TestMachinesDBus(machineslib.TestMachines):
         if self.machine.image == "ubuntu-stable":
             self.machine.execute("chmod o+rwx /run/libvirt/libvirt-sock")
 
+    def testAutostart(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-autostart-checkbox")
+
+        b.set_checked("#vm-subVmTest1-autostart-checkbox", True) # we dont not initial state of checkbox, so set it to checked
+        b.wait_present("#vm-subVmTest1-autostart-checkbox:checked")
```

Comment:
This only checks the checkbox, which by itself is not very interesting. Can this call `virsh` to ensure that the VM is actually configured for auto-starting?

Reply:
Done.